girder/api/docs.py

```diff
@@ -112,13 +112,25 @@ def removeRouteDocs(resource, route, method, info, handler):
                 del routes[resource]
 
 
-def addModel(name, model):
+def addModel(resource, name, model):
     """
-    This is called to add a model to the swagger documentation.
+    Add a model to the Swagger documentation.
 
+    :param resource: The type of existing resource to add the model to.
+    :param resource: str
     :param name: The name of the model.
     :type name: str
     :param model: The model to add.
     :type model: dict
```

Comment:
So there is a bit of a problem with this change ...

Given the following test class, which is our current pattern for adding documentation. The addModel(...) will be called before route(...) is called that adds the resource to routes, so we get an exception say the resource type is not known .... We could move the addModel(...) call to the constructor but that feels a little icky to me.

```python

class ModelResource(Resource):
    def __init__(self):
        self.resourceName = 'model'
        self.route('POST', (), self.hasModel)


    @access.public
    def hasModel(self, params):
        pass

    addModel('model', 'Body', {
        'id': 'Body',
        'require': 'bob',
        'properties': {
            'bob': {
                'type': 'array',
                'items': {
                    'type': 'integer'
                }
            }
        }
    })

    hasModel.description = describe.Description('What a model') \
        .param('body', 'Where its at!', dataType='Body', required=True,
               paramType='body')
```